main.rb

```diff
@@ -52,15 +52,17 @@ def insert_student(app, age)
   print 'Name [Input the name]: '
   name = gets.chomp
   print 'Has parent permission? [Y/N]: '
-  parent_permission = gets.chomp
-  case parent_permission
+  pp = gets.chomp
+  case pp
   when 'Y'
-    app.create_student(name, age, nil, parent_permission: true)
+    pp = true
   when 'N'
-    app.create_student(name, age, nil, parent_permission: false)
+    pp = false
   else
     puts 'Invalid choice. Please try again'
+    return
   end
+  app.create_student(name, age, nil, parent_permission: pp)
   puts "\nPerson created successfully\n"
 end
```

Comment:
- [ ] As a requirement your `main.rb` is only to start your code and ask the user for the initial choice of options (or to quit) but in your case, you add all your other methods which should be inside the `app.rb` file so kildy remove all the following methods and add these methods in `app.rb` file :slightly_smiling_face:  

- create_student
- insert_teacher
- age_zero?
- insert_person
- insert_book
- insert_rental
- show_rentals_by_id
- show_books
- show_people
- show_books_with_index
- show_people_with_index